set_dark.sh

```diff
@@ -15,6 +16,6 @@ fi
 
 PROFILE=${1:-Default}
 
-gconftool-2 -s -t string /apps/guake/style/background/color `cat $dir/colors/$theme/base03`
-gconftool-2 -s -t string /apps/guake/style/font/color `cat $dir/colors/$theme/base0`
-gconftool-2 -s -t string /apps/guake/style/font/palette `cat $dir/colors/$theme/palette`
+PALETTE_STR=$(cat $dir/colors/$theme/palette_dark)
+PALETTE_STR_GCONF="\"${PALETTE_STR}\""
+dconf write "/apps/guake/style/font/palette" $PALETTE_STR_GCONF
```

Comment:
Here you can simply make it:
```bash
dconf write "/apps/guake/style/font/palette" "$(cat $dir/colors/$theme/palette_dark)"
```

Reply:
I found that the shell stripped the quote marks, while guake requires the value of `palette` to actually include quote marks at the start and end. I wrote the script to make the need (and source of) this strange requirement clear.